showCrime/conf/wait-for-it.sh

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env bash
+# SOURCE: https://github.com/vishnubob/wait-for-it
+#
+#   Use this script to test if a given TCP host/port are available
```

Comment:
IMHO, I don't like how the script is written and seems unnecessarily complex for what it does. I usually just roll my own. It's fine for now though.

Reply:
This is the script recommended by many in the Docker community. It was open source and available. Feel free to replace it later.